graph-db/serverless/src/utils/cmr/indexCmrCollection.js

```diff
@@ -25,6 +26,9 @@ export const indexCmrCollection = async (collection, gremlinConnection) => {
     }
   } = collection
 
+  // delete the collection first so that we can clean up its related documentation vertices
+  await deleteCmrCollection(conceptId, gremlinConnection)
```

Comment:
This may make bootstrapping quite a bit more expensive, which could result in that lambda timing out. It would only be a one-time problem, though